spec/inputs/syslog_spec.rb

```diff
@@ -1,5 +1,5 @@
 # encoding: utf-8
-require "logstash/devutils/rspec/spec_helper"
+require_relative "../spec_helper"
```

Comment:
I not sure I understand, we do a `require "logstash/devutils/rspec/spec_helper"` in all plugins !?